Anime-App/Feature/Anime/NetworkingManager/NetworkingManager.swift

```diff
@@ -35,7 +35,7 @@ class NetworkingManager {
         let request = URLRequest(url: url)
         
         let dataTask = URLSession.shared.dataTask(with: request) { data, response, error in
-            DispatchQueue.main.async {
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.9) {
```

Comment:
Don't do this!!!, revert this back to what is was. What you're doing here is making every network request have an intentional 0.9 delay which will slow down your app. If you want to delay hiding the spinner you should do this in the view controller. This shouldn't have code to force it to be slow...